test/test_fcntl.rs

```diff
@@ -805,3 +805,22 @@ fn test_f_transferextents() {
         .expect("transferextents failed");
     assert_ne!(res, -1);
 }
+
+#[cfg(target_os = "freebsd")]
+#[test]
+fn test_f_readahead() {
+    use nix::fcntl::*;
+
+    let tmp = NamedTempFile::new().unwrap();
+    // With TMPDIR set with UFS, the vnode name is not entered
+    // into the name cache thus path is always empty.
+    // Therefore, we reopen the tempfile a second time for the test
+    // to pass.
```

Comment:
Why would an empty path fail our test?

Reply:
ah nvm I think this is copy paste from another test..